src/benchmarks/gc/src/exec/GCPerfSim/GCPerfSim.cs

```diff
@@ -1035,7 +1187,7 @@ public CharSpan TakeWord()
     private static bool IsDigit(char c) =>
         '0' <= c && c <= '9';
     private static bool IsDigitOrDot(char c) =>
```

Comment:
nit - since you added a comma, perhaps call it `IsDigitOrDecimalSeparator`?

Reply:
Good point - thanks!